src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt

```diff
@@ -15,13 +15,19 @@ class AuroraVolumeMapperV1(val applicationFiles: List<AuroraConfigFile>) {
     val mountHandlers = createMountHandlers()
     val configHandlers = applicationFiles.findConfigFieldHandlers()
 
-    val handlers = configHandlers + mountHandlers + listOf(AuroraConfigFieldHandler("secretVault"))
+    val handlers = configHandlers + mountHandlers + listOf(
+            AuroraConfigFieldHandler("secretVault"),
+            AuroraConfigFieldHandler("secretVaultKeys")
+    )
 
 
     fun createAuroraVolume(auroraConfigFields: AuroraConfigFields): AuroraVolume {
 
         return AuroraVolume(
                 secretVaultName = auroraConfigFields.extractOrNull("secretVault"),
+                secretVaultKeys = auroraConfigFields.extractOrNull<String>("secretVaultKeys")?.let {
```

Comment:
Her mener jeg (korriger meg hvis jeg tar feil, @bjartek ;) at vi har "landet" på å la metodekall og lambdaparameter på samme linje;

secretVaultKeys = auroraConfigFields.extractOrNull<String>("secretVaultKeys")
  ?.let { it.split(",").map { key -> key.trim() } } 
  ?: listOf(),

Reply:
Det stemmer, hvis det er plass til alt på en linje. Jeg tror egentlig vi trenger å lage en liten style guide. Det er jo flere andre plasser i koden hvor det ikke er slik.